core/src/main/java/org/mskcc/cbio/portal/model/GeneticAlterationType.java

```diff
@@ -36,8 +36,8 @@
 // don't forget to change the other one too
 public enum GeneticAlterationType {
     MUTATION_EXTENDED,
+    STRUCTURAL_VARIATION,
```

Comment:
this was already there. Please remove this cosmetic change.

Reply:
Done